web/themes/dcamp_2018_landing_theme/dcamp_2018_landing_theme.theme

```diff
@@ -10,5 +13,61 @@
  * @see https://www.drupal.org/node/2611246
  */
 function dcamp_2018_landing_theme_preprocess(&$variables, $hook) {
-  $variables['path_to_dcamp_2018_landing_theme'] = base_path() . drupal_get_path('theme', 'dcamp_2018_landing_theme');
+  $variables['path_to_dcamp_2018_theme'] = base_path() . drupal_get_path('theme', 'dcamp_2018_landing_theme');
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ *
+ * Adds the list of individual sponsors to the sponsors views block.
+ */
+function dcamp_2018_landing_theme_preprocess_frontpage(&$variables) {
```

Comment:
We don't need this yet, right?

Reply:
Let's remove this hook because it refers to logic that is not available yet (the attendees module will be installed in the next phase).

Reply:
ok, what you see to spare, eliminate it